vcpkg/get_started/overview.md

```diff
@@ -4,12 +4,93 @@ description: Learn about how the vcpkg C/C++ package manager can help you acquir
+
+### Manifests
+
+You can declare your direct dependencies and add optional features or version constraints in a [manifest file](../users/manifests.md).
+
+### Versioning
+
+vcpkg has a unique way of handling [package versions](../users/versioning.concepts.md). Your manifest file can reference a single, baseline version set by default. The baseline references a combination of ports, each at a specific version, that have been tested for compatibility against each other. Furthermore, you can set a minimum version for a package or override a package version to get an exact match.
+
+### Registries
+
+A [registry](../users/registries.md) is a catalog of ports and available versions that a vcpkg user can install. vcpkg provides a public registry of open-source libraries by default, but you can also define your own registries for custom libraries.
+
+### Asset caching
+
+[Asset caching](../users/assetcaching.md) allows vcpkg to use download-mirrors to upload and restore assets such as source code and build tools. This is useful for continuity of business, especially for open-source dependencies that have assets on third party servers. This also provides the developer protection in case upstream assets are tampered with unexpectedly.
 
 ## vcpkg compared to other package managers
 
-## Get started
+### vcpkg compared to NuGet
+
+NuGet is a .NET package manager that has often been used for C/C++ development, particularly for MSBuild solutions that also contain .NET projects. As a general rule, the Microsoft C++ team does not recommend using NuGet for C/C++ development, as NuGet has several limitations:
+
+- **Compilation flavors**. As NuGet cannot build packages from source on the fly, it is necessary to provide prebuilt binaries to match all possible Application Binary Interface (ABI) restrictions for all users. The user is responsible for building packages themselves and ensuring this is done correctly. It is also difficult to distinguish between binaries due to the lack of relevant metadata, resulting in the user having to do quirky things like putting architecture, operating system, and compiler information in the package name, without being able to enforce such constraints during package acquisition.
+- **Binary vs. source**. Very closely tied to the first point, NuGet is designed from the ground up to provide relatively small, prebuilt binaries. Due to the nature of native code, developers need to have access to the source code to ensure ABI compatibility, performance, integrity, and debuggability.
+- **Versioning limitations**. While NuGet packages are versioned, it is up to the user to decide on appropriate versions of packages to take dependencies on. The pressure is on the user to solve version conflicts between dependencies and the consuming project.
```

Comment:
This point looks like a positive from a certain perspective: "Oh, I'm a smart, advanced user, I like having lots of control! Does vcpkg not let me decided on the appropriate versions of packages to use?" Our versioning system is slightly easier to use, but I wouldn't describe NuGet as having "Versioning limitations".